src/components/utilities/getLicenseAmendments.js

```diff
@@ -20,7 +21,7 @@ export default (linkedLicense, status) => {
     });
 
   if (status !== undefined) {
-    return amendments.filter(a => get(a, 'statusForThisAgreement.value', null) === status);
+    return amendments.filter(a => get(a, 'statusForThisAgreement.value', statuses.UNSET) === status);
```

Comment:
Yeah this is way more obvious, I love it. Thanks!

Reply:
Nice one 😄